tests/backends/test_creation.py

```diff
@@ -88,3 +94,74 @@ def test_sql_table_creation_suffix_with_template(self):
     def test_sql_table_creation_suffix_with_encoding_and_template(self):
         settings = dict(CHARSET='UTF8', TEMPLATE='template0')
         self.check_sql_table_creation_suffix(settings, '''WITH ENCODING 'UTF8' TEMPLATE "template0"''')
+
+
+@unittest.skipUnless(connection.vendor == 'oracle', "Oracle specific tests")
```

Comment:
I guess we have poor organization where some tests are organized by class (test_creation/features) and others are organized by database (test_mysql). I'm not requiring a change here but it would be nice to think about how we want to do this going forward.